lua/nvim-tree/view.lua

```diff
@@ -133,7 +133,10 @@ local function setup_tabpage(tabpage)
 end
 
 local function set_window_options_and_buffer()
-  pcall(vim.cmd, "buffer " .. M.get_bufnr())
+  pcall(function()
+    vim.cmd("buffer " .. M.get_bufnr())
+  end)
```

Comment:
Could we revert this one as well? I'm the idiot who put it in but it's no longer necessary.

This function needs a rewrite anyway - we need to listen for failures and handle them.